.github/workflows/close-stale-prs.yml

```diff
@@ -0,0 +1,18 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '0 12 * * 1,2,3,4,5'
+  workflow_dispatch:
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v5
+        with:
+          days-before-pr-stale: 20
+          days-before-pr-close: 5
+          stale-pr-message: 'This PR is stale because it has been open 20 days with no activity. It will be closed in 5 days unless you remove the `stale` label or comment.'
+          close-pr-message: 'This PR was closed because it has been marked stale for 5 days with no activity.'
+          exempt-pr-labels: 'stale-exempt'
```

Comment:
Just maybe create this label if we haven't already!

Reply:
ohhh right! ty